tldw_Server_API/app/core/DB_Management/Kanban_DB.py

```
@@ -115,37 +165,63 @@ class KanbanDB:
-        conn = sqlite3.connect(self.db_path, timeout=30, isolation_level=None)
+    def _configure_connection(self, conn: sqlite3.Connection, *, enable_wal: bool = True) -> None:
         conn.row_factory = sqlite3.Row
         try:
-            conn.execute("PRAGMA journal_mode=WAL")
+            if enable_wal:
+                conn.execute("PRAGMA journal_mode=WAL")
             conn.execute("PRAGMA foreign_keys=ON")
             conn.execute("PRAGMA busy_timeout=30000")
             conn.execute("PRAGMA synchronous=NORMAL")
             conn.execute("PRAGMA cache_size=-64000")  # 64MB cache
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)